open-metadata-implementation/adapters/open-connectors/repository-services-connectors/open-metadata-collection-store-connectors/inmemory-repository-connector/src/main/java/org/odpi/openmetadata/adapters/repositoryservices/inmemory/repositoryconnector/InMemoryOMRSMetadataCollection.java

```diff
@@ -4919,7 +4920,7 @@ public void refreshRelationshipReferenceCopy(String userId,
      *                                  entity.
      * @throws PagingErrorException the paging/sequencing parameters are set up incorrectly.
      */
-    private List<EntityDetail>    formatEntityResults(List<EntityDetail>   fullResults,
+    protected static List<EntityDetail>    formatEntityResults(List<EntityDetail>   fullResults,
```

Comment:
No unnecessary use of statics in Egeria modules - this is because it reduces the flexiblity of the code.